src/plugins/discover/public/application/angular/discover.js

```diff
@@ -501,7 +503,8 @@ function discoverController($route, $scope) {
     timefilter.getAutoRefreshFetch$().pipe(
       tap((done) => {
         autoRefreshDoneCb = done;
-      })
+      }),
+      filter(() => $scope.fetchStatus !== fetchStatuses.LOADING)
```

Comment:
QQ: didn't test but wouldn't it be cleaner to first `filter` and then `tab`, or doesn't it matter in this case?

Reply:
We need `tap` first to get a handle of that `done` so that when pending fetch completes we notify `timefilter` that it is OK to start the next auto-refresh loop. 

In case `tap` is after `filter` then we won't get a handle of `autoRefreshDoneCb` so we won't call it when requests completes and next auto-refresh loop won't start

Reply:
@Dosant thx for the clarification 👍